File: packages/cli/tests/commands/template/fixtures/multiple-env/expected/typescript/build.prod.ts
import { Template, defaultBuildLogger, handleBuildError } from 'e2b'
import { template } from './template'

async function main() {
  await Template.build(template, 'env-test', {
    onBuildLogs: defaultBuildLogger(),
  });
}

main().catch(handleBuildError);

File: packages/cli/tests/commands/template/fixtures/complex-python/expected/python-sync/build_prod.py
from e2b import Template, default_build_logger, handle_build_error
from template import template


if __name__ == "__main__":
    try:
        Template.build(
            template,
            "complex-python-app",
            on_build_logs=default_build_logger(),
        )
    except Exception as err:
        handle_build_error(err)